core/zms/src/main/rdl/Domain.rdli

```diff
@@ -319,4 +319,20 @@ resource AuthHistoryDependencies GET "/domain/{domainName}/history/auth" {
         ResourceError UNAUTHORIZED;
         ResourceError TOO_MANY_REQUESTS;
     }
+}
+
+type ExpiredMembers Struct {
+    Array<MemberName> expiredRoleMembers; //list of deleted expired role members names
+    Array<GroupMemberName> expiredGroupMembers; //list of deleted expired groups members names
+}
+
+// Delete expired principals
+resource ExpiredMembers DELETE "/expired-members?purgeResources={purgeResources}" {
+    Int32 purgeResources (optional); //defining which resources will be purged. by default all resources will be purged
+    expected NO_CONTENT;
```

Comment:
I think here it would be useful to return OK. NO_CONTENT and use the support we added for UI to ask the list of expired members to be returned or not.